.travis.yml

```diff
@@ -5,5 +5,5 @@ env:
     - DOCKER_IMAGE=fedora:24
     - DOCKER_IMAGE=fedora:25
     - DOCKER_IMAGE=fedora:rawhide
```

Comment:
Do you mean replacing `fedora:24` and `fedora:25` with `fedora:latest`? I used `fedora:latest` in Makefile.docker just to avoid the need of changing the value every time new Fedora is released.